plugins/secretstores/http/http.go

```diff
@@ -47,6 +47,8 @@ func (h *HTTP) SampleConfig() string {
 
 func (h *HTTP) Init() error {
 	ctx := context.Background()
+
+	// FIXME: on reload, idle connections from this client may be kept open
```

Comment:
I don't see any hint of a stop method being available for secretstore plugins, so have left this as a FIXME. Happy to change it if I'm wrong.

Reply:
I think you can set the idle-timeout for the client here.

Reply:
OK, I've set this to five minutes.